Documentation/ceph-osd-mgmt.md

```diff
@@ -70,14 +70,13 @@ in the toolbox may show which OSD is `down`.
 2. Mark the OSD as `out` if not already marked as such by Ceph. This signals Ceph to start moving (backfilling) the data that was on that OSD to another OSD.
    - `ceph osd out osd.<ID>` (for example if the OSD ID is 23 this would be `ceph osd out osd.23`)
 3. Wait for the data to finish backfilling to other OSDs.
-   - `ceph status` will indicate the backfilling is done when all of the PGs are `active+clean`.
-4. Remove the disk from the node.
-5. Update your CephCluster CR such that the operator won't create an OSD on the device anymore.
+   - `ceph status` will indicate the backfilling is done when all of the PGs are `active+clean`. It's safe to remove the disk after that.
```

Comment:
```suggestion
   - `ceph status` will indicate the backfilling is done when all of the PGs are `active+clean`. If desired, it's safe to remove the disk after that.
```

Reply:
Thanks, fixed.

Reply:
Have you pushed yet?

Reply:
I just pushed. There was a problem with my github authentication.